Scripts/Runtime/Logging/.CSProject/UnityLogHandler.cs

```diff
@@ -24,7 +24,11 @@ public class UnityLogHandler : ILogHandler
             string callerName,
             int callerLine)
         {
-            message = $"({Path.GetFileNameWithoutExtension(callerPath)}:{callerLine}|{callerName}) {message}";
+            string callerFile = Path.GetFileNameWithoutExtension(callerPath);
+
+            string context = (callerLine > 0 ? $"{callerFile}:{callerLine}|{callerName}" : $"{callerFile}|{callerName}");
+
+            message = $"({context}) {message}";
```

Comment:
let's add the message in one step to avoid the extra string allocation.

```suggestion
            if(callerLine > 0)
            {
               message = $"({callerFile}:{callerLine}|{callerName}) {message}"
            }
            else
            {
               message = $"({callerFile}|{callerName}) {message}"
            }
```